crates/runtime/src/traphandlers/macos_aarch64.S

```diff
@@ -0,0 +1,60 @@
+.section __TEXT,__text,regular,pure_instructions
+
+#define GLOBL(fnname) .globl _##fnname
+#define HIDDEN(fnname) .private_extern _##fnname
+#define TYPE(fnname)
+#define FUNCTION(fnname) _##fnname
+#define SIZE(fnname)
+
+HIDDEN(wasmtime_macos_aarch64_unwind)
+GLOBL(wasmtime_macos_aarch64_unwind)
+
+// This small stub helps connect the native code and the wasm call frames when
+// unwinding a stack. This must be kept in close sync with macos.rs in the same
+// directory.
+
+// The signature of this function is:
+// wasmtime_macos_aarch64_unwind(faulting_pc, setjmp_buf, unwind_fn)
```

Comment:
eh I wouldn't really say that this function has a signature  because it's not natively callable, but this could perhaps describe the input register expectations.